services/app-api/serverless.yml

```diff
@@ -61,10 +62,12 @@ provider:
   name: aws
   runtime: nodejs12.x
   region: us-east-1
+  logs:
+    restApi: true
```

Comment:
With the advent of serverless-iam-helper plugin, this is now possible.  And it's possible to do it in an idempotent fashion with the serverless-idempotency-helper plugin.